wordpress-mobile/WordPress-iOS
# Issues/reader multitasking

This PR includes a few layout tweaks to improve support for iOS 9 Multitasking in the Reader list, detail and comments, and also in the my sites Post list. 

Repeat this test with the Reader List, Reader Detail, Reader Comments, and My Sites Post List

To test:
1. Launch the app on an iPad Air 2 (Simulator)
2. Open the feature your testing
3. Try rotating the device
4. Switch to Safari
5. Try "Slide Over". Attempt to rotate the device
6. Resize WPiOS's screen, so that it covers 50% of the screen
7. Try rotating again

Note: DTCoreText is notoriously glitchy in the comment cells running in the Simulator.  If you see any bizarre artifacting its a known issue and (so far) has not reproduced on a device. (Can't wait to kiwf!) 

@jleandroperez Since you're already fussed with multitasking for Notifications would you mind reviewing this one? 

Needs review: @jleandroperez
